Undeprecate v2 metrics (#1375)
minio/docs: source/operations/monitoring/metrics-and-alerts.rst

   :local:
   :depth: 2

.. admonition:: Metrics version 3
   :class: note

> 

Reviewer: I want to say something at the top of the page like "This page covers version 3 of the metrics, introduced in blah blah. For version 2 of the metrics, see bling." Maybe even outside of the admonition, since people often skip those.

Author: rearranged some words so there's a bit of an introduction

>    Starting with MinIO Server :minio-release:`RELEASE.2024-07-15T19-02-30Z` and MinIO Client :mc-release:`RELEASE.2024-07-11T18-01-28Z`, metrics version 3 provides additional metrics scraping endpoints.

Reviewer: ```suggestion
   Starting with MinIO Server :minio-release:`RELEASE.2024-07-15T19-02-30Z` and MinIO Client :mc-release:`RELEASE.2024-07-11T18-01-28Z`, metrics version 3 provides additional endpoints.
```

Just trying to limit the number of uses of the word `metric`.

   MinIO recommends version 3 for new deployments.

>    Existing deployments can continue to use :ref:`metrics version 2 <minio-metrics-v2>` and :ref:`the v2 Grafana dashboards <minio-grafana>`.

Reviewer: ```suggestion
   Existing deployments can continue to use version 2 :ref:`metrics <minio-metrics-v2>` and :ref:`Grafana dashboards <minio-grafana>`.
```

MinIO publishes cluster and node metrics using the :prometheus-docs:`Prometheus Data Model <concepts/data_model/>`.
You can use any scraping tool to pull metrics data from MinIO for further analysis and alerting.

> 

Reviewer: Maybe another h2 here?

```
Version 3 Endpoints
-------------------
```

Author: yeah I think that makes sense 👍🏻

For metrics version 3, all metrics are available under the base ``/minio/metrics/v3`` endpoint.

> You can scrape the base endpoint to collect all metrics in a single scraping operation, or append an optional path to scrape a single category.

Reviewer: ```suggestion
You can scrape the base endpoint to collect all metrics in a single operation, or append an optional path to return a specific category.
```
